components/Navbar/NavItem.js

```diff
@@ -0,0 +1,24 @@
+import NextLink from "next/link";
+import { Button } from "@chakra-ui/react";
+const Pages = ["Home", "Contact", "Donores", "About"];
+function index() {
+  return Pages.map((item, i) => (
+    <div key={item}>
+      <NextLink href={item === "Home" ? "/" : `/${item}`} passHref>
```

Comment:
Are we rendering the Pages and their routes here? Is that way we have "Home" ? "/" if so, then create an object for all the pages something like 
```
const pages = [
{title="Home", route="/"},
{title="About", route="/about"},
]
```